test/unittests/api_test.cpp

```diff
@@ -18,8 +18,8 @@ namespace
 ExecuteFunction function_returning_value(Value value)
 {
     static constexpr HostFunctionPtr func = [](std::any& host_context, Instance&, const Value*,
-                                                int) -> ExecutionResult {
-        return std::any_cast<Value>(host_context);
+                                                int) noexcept -> ExecutionResult {
+        return *std::any_cast<Value>(&host_context);
```

Comment:
`any_cast` taking a reference returns error by throwing `bad_cast` exception.
Here it's changed to `any_cast` taking a pointer, which returns `nullptr` in case of error.

Reply:
Ah okay. Though this could have been done as reference in the first place in #716.